### DISCUSSION TITLE
Remarks/improvements on Dutch translation

### DISCUSSION THREAD

**Discussion by Benny7587** [rustdesk/rustdesk#3253] [General]
I see that the latest nightly build hs a dutch translation. Not sure if this is an automatic Google translate, but when it is a manual translate i have the following improvemnets (imho):

**Main window:**
'Klaar' a better translation would be 'Gereed' or 'Verbonden' (which means connected).

**First Tab:** 'Recente behandelingen' > 'Recente sessies'.

**Settings - Security:**
'Eenmalig wachtwoord lengre:' > 'Lengte eenmalig wachtwoord:' 

**Settings - About:**
'Gedaan met het hart in deze chaotische wereld!' > 'Met liefde gemaakt in deze chaotische wereld!'

**Comment by rustdesk** [comment]
It is not google translate.

**Reply by rustdesk** [reply]
@FastAct

**Comment by Benny7587** [comment]
One more remark about the Dutch translation:
The translation above the Install button is very extensive. I think the following short version would suffice at least for me ;)

'Als RustDesk aan de remote kant niet goed werkt vanwege UAC, klik dan op onderstaande knop om RustDesk aan de remote kant te installeren.'
